examples/Tutorial_custom.ipynb

```diff
@@ -47,18 +47,22 @@
     "The Simularium custom data Converter consumes spatiotemporal data from any source. It requires a `CustomData` object as parameter.\n",
     "\n",
     "`CustomData` contains the following:\n",
-    "* **spatialUnitFactorMeters** : *float*\n",
-    "    * A float multiplier needed to convert spatial data (including positions, radii, and box size) to meters\n",
     "* **box_size** : *np.ndarray (shape = \\[3\\])*\n",
     "    * A numpy ndarray containing the XYZ dimensions of the simulation bounding volume\n",
     "* **agent_data** : `AgentData`\n",
     "    * An object containing data for each agent at each timestep\n",
+    "* **time_unit_factor_seconds** : *float (optional)*\n",
+    "    * A float multiplier needed to convert temporal data (e.g. timeStepSize) to seconds\n",
+    "    * Default: 1.0 (seconds)\n",
+    "* **spatialUnitFactorMeters** : *float (optional)*\n",
```

Comment:
Does this need to be `spatial_unit_factor_meters` instead? Seeing snake case elsewhere